tests/models/test_lora_layers.py

```diff
@@ -43,15 +44,35 @@ def create_unet_lora_layers(unet: nn.Module):
     return lora_attn_procs, unet_lora_layers
 
 
-def create_text_encoder_lora_layers(text_encoder: nn.Module):
+def create_text_encoder_lora_attn_procs(text_encoder: nn.Module):
     text_lora_attn_procs = {}
     for name, module in text_encoder.named_modules():
-        if any(x in name for x in TEXT_ENCODER_TARGET_MODULES):
-            text_lora_attn_procs[name] = LoRAAttnProcessor(hidden_size=module.out_features, cross_attention_dim=None)
+        if name.endswith(TEXT_ENCODER_ATTN_MODULE):
```

Comment:
Sorry if question some dumb things, didn't lora in text_encoder also uses `mlp`?
![image](https://github.com/huggingface/diffusers/assets/7768370/4a1530b5-762d-4681-aab7-334143839692)
![image](https://github.com/huggingface/diffusers/assets/7768370/28fbfe90-57b0-4706-81e8-b26750912ad0)

Reply:
I have same question here
![20230529173440](https://github.com/huggingface/diffusers/assets/8824848/ecfa6b43-cbe4-4294-97bd-44f69678fefe)

Reply:
Correct, there is also LoRA for ClipMLP. However, we have decided to keep it out of the scope of this PR. I plan to work on it in a separate PR. https://github.com/huggingface/diffusers/pull/3437#issue-1710465786